ads_common/lib/ads_common/auth/oauth2_handler.rb

```diff
@@ -86,6 +86,11 @@ def get_token(credentials = nil, force_refresh = false)
         token = super(credentials)
         token = refresh_token! if !@client.nil? &&
             (force_refresh || @client.expired?)
+
+        if token.nil?
+          raise AuthError.new("Unable to get token due to missing oauth information.  Check account settings.")
+        end
```

Comment:
I'd use an else, rather than "falling through" to the return. Also, return keyword is not required.